src/main/java/com/github/monster860/fastdmm/FastDMM.java

```diff
@@ -385,7 +385,8 @@ public DMI getDmi(String name, boolean doInitGL) {
 			DMI dmi = null;
 			try {
 				if(name != null && name.trim().length() > 0) {
-					dmi = new DMI(new File(dme.getParentFile(), Util.separatorsToSystem(name)));
+					dmi = new DMI(new File(dme.getParentFile(),
+					                       objTree.filePath(Util.separatorsToSystem(name))));
```

Comment:
Really this is a discussion for coding styles which we haven't established (I've seen mixed types of `if (` and `if(`